hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -261,6 +261,10 @@ private ContainerCommandResponseProto dispatchRequest(
         // If container does not exist, create one for WriteChunk and
         // PutSmallFile request
         responseProto = createContainer(msg);
+        metrics.incContainerOpsMetrics(Type.CreateContainer);
+        metrics.incContainerOpsLatencies(cmdType,
```

Comment:
```suggestion
        metrics.incContainerOpsLatencies(Type.CreateContainer,
```

Reply:
Thank you Attila!! (sorry I missed to change it..